AWS/extract_lambda/bobae_extract.py

```
@@ -0,0 +1,159 @@
+        InvocationType="Event",
+        Payload=json.dumps(log_payload)
+    )
+
+def lambda_handler(event, context):
+    html_data = {}
+    print(f"📆 크롤링 기간: {END_TIME} ~ {TODAY}")
+    
+    # Step Function에서 전달된 키워드 가져오기
+    keywords = event.get("keywords", [])
+    if not keywords:
+        log_error("lambda_handler", url, "키워드 리스트가 입력되지 않음")
```

Ruff (0.8.2)

Undefined name `url`

(F821)